### ISSUE TITLE
Probable infinite loop for ./Netlist_to_graph/Parsers/SFLL_Verilog_to_graph.pl

### ISSUE THREAD

**Issue by tgourav** [DfX-NYUAD/GNNUnlock#2]
I am running the "./Netlist_to_graph/Parsers/SFLL_Verilog_to_graph.pl" script for SFLL benchmarks and the program keeps on running for more than 20 hours and does not stop. Script to convert ANTISAT dataset to graph is working fine and It finishes within 10 minutes. Could you please resolve this issue? 
I followed the steps mentioned in README.md
Steps I took to run are as follows:

1. Uncommented line 8 and modified it in file Parsers/SFLL_Verilog_to_graph.pl
2. cd ./Netlist_to_graph/Circuits_datasets/SFLL_DATASET_c7552
3. cp ../../Parsers/graph_parser.py ./
2. Ran the following command $ perl ../../Parsers/SFLL_Verilog_to_graph.pl -i ../../Circuits_datasets/SFLL_DATASET_c7552 > log.txt

**Comment by lilasrahis** [comment]
Hello,

Are you running the code from Circuits_datasets/SFLL_DATASET_c7552 directory? You need to run it from Graphs_datasets/SFLL_DATASET_c7552/. So you just unzip Circuits_datasets/SFLL_DATASET_c7552, but you don't run anything from there. The exact steps are explained in README.

**Comment by tgourav** [comment]
Thanks for the response. It works now.

I faced the following error when I run the following training command:
~/GNNUnlock/GraphSAINT$ python -m graphsaint.tensorflow_version.train --data_prefix ../Netlist_to_graph/Graphs_data
sets/SFLL_DATASET_c7552 --train_config ../SFLL_params.yml --gpu 1 > log_training.txt                                                             

WARNING:tensorflow:From /data/tgourav/GNNUnlock/GraphSAINT/graphsaint/tensorflow_version/model.py:127: softmax_cross_entropy_with_logits (from te
nsorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See `tf.nn.softmax_cross_entropy_with_logits_v2`.

2021-10-08 12:25:17.813543: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was n
ot compiled to use: SSE4.1 SSE4.2 AVX
/data/tgourav/GNNUnlock/GraphSAINT/graphsaint/tensorflow_version/minibatch.py:136: RuntimeWarning: invalid value encountered in true_divide
  val = np.clip(self.norm_loss_train[v]/self.norm_aggr_train[i_s:i_e], 0, 1e4)
2021-10-08 12:25:30.300308: E tensorflow/core/common_runtime/bfc_allocator.cc:373] tried to deallocate nullptr
2021-10-08 12:25:30.300380: E tensorflow/core/common_runtime/bfc_allocator.cc:373] tried to deallocate nullptr
2021-10-08 12:25:30.300410: E tensorflow/core/common_runtime/bfc_allocator.cc:373] tried to deallocate nullptr

**Comment by lilasrahis** [comment]
I think the code is trying to "divide by zero" or "divide by NaN". Are you sure the files in SFLL_DATASET_c7552 are not empty?
